tools/compress_json.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+
+import json
+import struct
+import sys
+import zlib
+
+if __name__ == '__main__':
+  fp = open(sys.argv[1])
```

Comment:
not sure if important for this kind of script, but fp is not explicitly closed. Since we depend on 2.6-2.7, maybe use a with statement instead?